ui/src/components/Sidebar.tsx

```diff
@@ -860,7 +863,17 @@ function ExportJSON() {
   );
   useEffect(() => {
     if (data?.exportJSON) {
-      download(data.exportJSON);
+      let element = document.createElement("a");
+      element.setAttribute(
+        "href",
+        "data:text/plain;charset=utf-8," + encodeURIComponent(data.exportJSON)
+      );
+      element.setAttribute("download", `${repoName || ""}.json`);
```

Comment:
Let's add date to the exported filename and keep consistent with the .ipynb export.

https://github.com/codepod-io/codepod/blob/554cdac415a9ce579aa6a0b81a8390b7927a91fe/ui/src/components/Sidebar.tsx#L891-L893

Reply:
Nevermind, I made this minor change. Mergin.